haystack/core/pipeline/breakpoint.py

```diff
@@ -248,6 +250,38 @@ def _save_pipeline_snapshot(pipeline_snapshot: PipelineSnapshot) -> PipelineSnap
     return pipeline_snapshot
 
 
+def _save_component_input(
+    component_inputs: dict[str, Any], component_name: str, visit_nr: int, out_path: Union[str, Path]
+) -> None:
```

Comment:
I think this is function is only needed for the other feature right? I don't see it being used elsewhere in this PR.